app/controllers/v1/contestants_controller.rb

```
@@ -7,10 +7,13 @@ class ContestantsController < ApplicationController
 
     # GET /v1/contestants.json
     def index
-      editition_id = if params.has_key?(:edition)
-        params[:editition]
+      if params.has_key?(:edition)
+        peditition_id = arams[:editition]
```

versiunea care am propus-o mai sus nu-ti da exceptie. O sa fie un silent failure mic.